src/main/java/bio/ferlab/fhir/FhavroConverter.java

```diff
@@ -37,22 +34,22 @@
     }
 
     @NotNull
-    public static <T extends BaseResource> T convertGenericRecordToResource(GenericRecord genericRecord, Schema schema, Class<T> type) {
-        return AvroFhirConverter.readGenericRecord(genericRecord, schema, type);
+    public static <T extends BaseResource> T convertGenericRecordToResource(GenericRecord genericRecord, Schema schema, String name) {
+        return AvroFhirConverter.readGenericRecord(genericRecord, schema, name);
     }
 
     @NotNull
     public static Schema loadSchema(String schemaName) {
         schemaName = ConverterUtils.formatSchemaName(schemaName);
 
-        URL resource = ClassLoader.getSystemClassLoader().getResource("schemas/" + schemaName);
-        if (resource == null) {
+        InputStream inputStream = ClassLoader.getSystemClassLoader().getResourceAsStream("schemas/" + schemaName);
```

Comment:
This is to ensure that the library can read its own schema when called from an external project (see: https://stackoverflow.com/questions/10144210/java-jar-file-use-resource-errors-uri-is-not-hierarchical)